src/main/java/org/folio/service/transactions/batch/BatchTransactionChecks.java

```diff
@@ -65,11 +67,12 @@ public static void sanityChecks(Batch batch) {
           checkTransfer(transaction);
         }
       });
+    checkPaymentsAndCreditsAmounts(batch);
   }
 
   public static void checkBudgetsAreActive(BatchTransactionHolder holder) {
     holder.getBudgets().forEach(budget -> {
-      if (budget.getBudgetStatus() != ACTIVE) {
+      if (!List.of(ACTIVE, PLANNED).contains(budget.getBudgetStatus())) {
```

Comment:
this change is needed because of some issue, or you just thinking that support of planned budgets was missed from beginning?

Reply:
It worked before because the allocation endpoint was used when creating allocations for planned budgets. Now that the batch endpoint is used for that, it needs to allow the creation of allocations for planned budgets (I didn't add a check to restrict that to allocations to avoid making the code more complex for an edge case).